CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## [Unreleased]
 
+## [1.7.34]
+
+- update tk-metrics to 1.2.2
```

Comment:
Can we add a comment here, that this is the public version of 1.2.1? Otherwise this changelog is literally _less_ useful than the commit history, and there's no point in having it (and we already had that debate 🙂). I find it useful to have a quick summary what changes the bumped dep brought along.

Reply:
well the changelog is always less useful than the commit history, it's just sometimes faster :P but yes, updated the changelog

Reply:
Fair enough haha